code/modules/antagonists/traitor/equipment/Malf_Modules.dm

```diff
@@ -196,8 +196,8 @@ GLOBAL_LIST_INIT(malf_modules, subtypesof(/datum/ai_module))
 		return
 	if (active || owner_AI.stat == DEAD)
 		return //prevent the AI from activating an already active doomsday or while they are dead
-	if (owner_AI.shunted)
-		return //prevent AI from activating doomsday while shunted, fucking abusers
+	if (!isturf(owner_AI.loc))
```

Comment:
why are you replacing the shunted check with the isturf() instead of checking both?

Reply:
when the AI is shunted, firstly it loses its hud (from testing), and secondly (according to the comment here) its loc should be the apc, which isnt a turf, so this check should be redundant
https://github.com/tgstation/tgstation/blob/f4b83b17a51e6cd42f99171709c4736d66455074/code/modules/mob/living/silicon/ai/ai.dm#L776-L784